policyengine_taxsim/config/variable_mappings.yaml

```diff
@@ -38,250 +54,412 @@ policyengine_to_taxsim:
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "21. General Tax Credit 1975-8"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v22:
     variable: ctc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "22. Child Tax Credit*17/22 98"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v23:
     variable: refundable_ctc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "23. Refundable Part"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v24:
     variable: cdcc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "24. Child Care Credit 1076+"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v25:
     variable: eitc
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "25. Earned Income Credit 1975"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v26:
     variable: amt_income
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "26. Alternative Min Income"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v27:
     variable: alternative_minimum_tax
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "27. AMT"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v28:
     variable: income_tax_before_refundable_credits
     implemented: true
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "28. Income Tax Before Credits \n Total Credits"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v29:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "29. FICA \n  Taxpayer share of FICA"
+    group_order: 3
+    full_text_group: "Federal Tax Calculation"
+    group_column: 2
   v30:
     variable: household_net_income
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "30. Household Income"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v31:
     variable: not_implemented_in_pe
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "31. Imputed Rent"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v32:
     variable: state_agi
     implemented: false
     idtl:
      - full: 2
      - full_text: 5
+    text_description: "32. AGI"
+    group_order: 4
+    full_text_group: "State Tax Calculation"
+    group_column: 2
   v33:
     variable: not_implemented_in_pe
```

Comment:
We have state exemptions - just not in one variable (naming varies) - is this a change in the US package that needs to happen?

Reply:
generally using 'state_exemptions' should produce a valid result in ~80% of the states